test/entry_set_test.rb

```diff
@@ -66,9 +66,17 @@ def test_entries
   end
 
   def test_find_entry
-    # by default, #find_entry is case-sensitive
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('name1'))
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('NaMe1', false))
```

Comment:
AGAIN. If you will set `case_insensitive_match ` it will be case insensitive for ALL gems what uses rubyzip. Even if it use new feature or not.